src/components/TextButton.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import styled, { CSSProperties } from 'styled-components'
+
+export interface textButtonInterface {
+    text: string
+    onClick: () => void
+    style?: CSSProperties
```

Comment:
So, I found out that we don't actually need this. When someone wants to override the styles of you component they can do this:
```typescript
const CustomTextButton = styled(CustomButton)`
    font-size: 3em;
`
```

And to do this is all in styled-components, no need to accept a style prop or pass anything down!

Reply:
Perfect, thanks!